instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -200,17 +242,23 @@ def get_or_create_headers():
                     SpanAttributes.HTTP_STATUS_CODE
                 ] = result.status_code
 
-                if result.raw is not None:
-                    version = getattr(result.raw, "version", None)
-                    if version:
-                        metric_labels[SpanAttributes.HTTP_FLAVOR] = (
-                            "1.1" if version == 11 else "1.0"
-                        )
+                metric_labels[
+                    SpanAttributes.NETWORK_PROTOCOL_NAME
```

Comment:
it's opt-in on metrics

Reply:
Got it. So, we should set the NETWORK_PROTOCOL_NAME network label only for http and http_dup?

Reply:
Removing and leaving opt-in as to-do